sns/parity_ops.h

```diff
@@ -25,11 +25,25 @@
 #ifndef __MOTR_SNS_PARITY_OPS_H__
 #define __MOTR_SNS_PARITY_OPS_H__
 
+#ifndef __KERNEL__
+#include <isa-l.h>
+#else
 #include "galois/galois.h"
+#endif /* __KERNEL__ */
```

Comment:
So, here, we will:

```

#ifdef HAVE_ISAL
#include <isa-l.h>
#else
#include "galois/galois.h"
#endif

Reply:
Yes. Also we need to check for __KERNEL__, because in kernel space we want to use Galois library. So it would be something like 
```
#if !defined(__KERNEL__) && defined(HAVE_ISAL)
#include <isa-l.h>
#else
#include "galois/galois.h"
#endif
```